wandering-warriors/main.py

```diff
@@ -27,9 +33,71 @@ class Ledger(Widget):
 class OperationsBar(Widget):
     pass
 
-
-class CuneiformDrawingInput(Widget):
-    pass
+def calculate_points(x1, y1, x2, y2, steps=1):
```

Comment:
Could we move this to a separate file? I don't know where this would go, but getting it out of here would be very nice.

Reply:
It might be more appropriate at the top of the file. I guess I could move it out, but idk if it's worth it for now.